CHANGELOG.md

```diff
@@ -3,6 +3,120 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+We'd like to offer a big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Use the bundling scripts as in [material-ui](https://github.com/mui/material-ui) repository (#3965) @flaviendelangle
```

Comment:
I think @m4theushw suggested adding "same" in here which I agree with. I've also added a small grammatical fix.

```suggestion
- 📦 Use the same bundling scripts as those in the [material-ui](https://github.com/mui/material-ui) repository (#3965) @flaviendelangle
```